test/corpus/jsx.txt

```diff
@@ -310,3 +310,20 @@ Spread props
             (value_identifier))))
       (jsx_closing_element
         (jsx_identifier)))))
+
+================================================================================
+Elements in pipes
+================================================================================
+
+<Comp />->Some
```

Comment:
A similar snippet was failing in the wild tests (but I forgot to add it to the list in the description.)

The problem was that pipe expressions only accept `primary_expression`s as their members, and JSX elements wasn't one. I fixed it by simply adding it to the list of `primary_expression`s. I looked at the grammar and this shouldn't have any negative repercussions.